src/components/Home.js

```diff
@@ -5,6 +5,7 @@ import React from 'react';
 import { Map, TileLayer, Marker, Popup, Circle } from 'react-leaflet';
 import type { LatLng } from 'react-leaflet/es/types';
 import { withStyles, Button } from 'material-ui';
+import { withRouter } from 'react-router-dom';
```

Comment:
seemed to be added but not used in new code, what is the purpose of it?

Reply:
as discuss with Patrick the purpose is to be used in further code he is right now developing.
It would be prettier to leave it out but as it is currently already in it can be left in it